docs/serverless/assistant/connect-to-vertex.mdx

```diff
@@ -0,0 +1,68 @@
+---
+slug: /serverless/security/connect-to-google-vertex
+title: Connect to Google Vertex
+description: Set up a Google Vertex LLM connector.
+tags: ["security", "overview", "get-started"]
+status: in review
+---
+
+# Connect to Google Vertex AI
```

Comment:
This line makes it look like there are two H1s on the page – **Connect to Google Vertex** and **Connect to Google Vertex AI**. We can probably remove this line.